### DISCUSSION TITLE
Why installing React Native?

### DISCUSSION THREAD

**Discussion by gxanshu** [firebase/firebase-js-sdk#7527] [Q&A] [answered]
i recently adding Firebase into my new React JS project and i as i noticed installing Firebase JS SDK from NPM it also installing `react-native` and `jsc-android`. i just want to know is it a bug or is there any use of react native in firebase sdk.
![Screenshot from 2023-08-04 23 52 09](https://github.com/firebase/firebase-js-sdk/assets/68537640/6cf9fa43-f646-4d64-96fa-3b878cb74339)

**Comment by ryanmio** [comment, marked as answer]
Not a bug, also not a problem. Firebase JS SDK includes support for React Native. 'react-native' and 'jsc-android' are installed but not used unless you're actually in a React Native environment. These packages won't affect your React JS project

**Reply by gxanshu** [reply]
Thanks

**Reply by hsubox76** [reply]
FYI we also reverted this in 10.3.0 https://firebase.google.com/support/release-notes/js#version_1030_-_august_22_2023
